core/validation/checks.py

```diff
@@ -11,121 +11,137 @@ class Check(ABC):
         sheet (str): The name of the sheet in the workbook.
         column (int): The column index of the cell to check.
         row (int): The row index of the cell to check.
-        expected_values (tuple or None): A tuple of expected values for the cell, or None if any value is acceptable.
+        expected_values (tuple): A tuple of expected values for the cell.
         error_message (str): The error message sent to the user if the check fails.
-        actual_value (str or None): The actual value retrieved from the cell on the ingested spreadsheet.
 
     Methods:
+        get_actual_value(workbook: dict[str, pd.DataFrame]) -> str:
+            Retrieve the actual value from the workbook.
         run(workbook: dict[str, pd.DataFrame]) -> bool:
             Execute the check on the provided workbook.
     """
 
-    def __init__(self, sheet: str, column: int, row: int, expected_values: tuple | None, error_message: str):
+    def __init__(self, sheet: str, column: int, row: int, expected_values: tuple, error_message: str):
         self.sheet = sheet
         self.column = column
         self.row = row
         self.expected_values = expected_values
         self.error_message = error_message
-        self.actual_value = None
 
-    def run(self, workbook: dict[str, pd.DataFrame]) -> bool:
-        self.actual_value = str(workbook[self.sheet].iloc[self.column][self.row]).strip()
-        if self.actual_value not in self.expected_values:
-            return False
-        return True
+    def get_actual_value(self, workbook: dict[str, pd.DataFrame]) -> str:
+        return str(workbook[self.sheet].iloc[self.column][self.row]).strip()
+
+    def run(self, workbook: dict[str, pd.DataFrame]) -> tuple[bool, str]:
+        if self.get_actual_value(workbook) not in self.expected_values:
+            return False, self.error_message
+        return True, ""
```

Comment:
We could pull `run` and `expected_values` out to `BasicCheck`? We could add `run` as an abstract method to this base class to enforce it's implementation in all sub classes.. Just a thought not a hard suggestion